src/components/table/data-table-multi-select-actions/trace-table-multi-select-action.tsx

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable */
+// @ts-nocheck
```

Comment:
We do not do this, please fix typing
```
/* eslint-disable */
// @ts-nocheck
```

Happy to help to get this merged, let me know if you'd prefer that